querybook/webapp/redux/board/action.ts

```diff
@@ -24,8 +21,11 @@ export const boardSchema = new schema.Entity('board', {
 
 function normalizeBoard(rawBoard: IBoardRaw) {
     const normalizedData = normalize(rawBoard, boardSchema);
-    const board: IBoardWithItemIds =
-        normalizedData.entities.board[normalizedData.result];
+    let board = normalizedData.entities.board[normalizedData.result];
+    board = {
+        ...board,
+        description: board.description as string,
```

Comment:
is board right now `IBoardWithItemIds`?
the typing can be determined as `const board: IBoardWithItemIds = normalizedData...`